packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -196,8 +194,8 @@ class OAuthController {
             return wsClient.notifyErr(res, wsClientId, providerConfigKey, connectionId, WSErrBuilder.TokenError());
         }
 
-        const sessionData = this.sessionStore[session.id]!;
-        sessionData.request_token_secret = tokenResult.request_token_secret;
+        session.request_token_secret = tokenResult.request_token_secret;
+        await oAuthSessionService.create(session);
```

Comment:
It's a bit weird that we are calling `oAuthSessionService.create(` twice for the same session in the OAuth1 case. 

We should probably either: 
- Separate the `create` and `update` methods (probably the cleanest solution)
- At least rename `create` to `createOrUpdate` or `upsert`

Reply:
I will go with createOrUpdate instead.

Reply:
Thanks!